core/src/main/resources/struts-default.xml

```diff
@@ -55,13 +55,18 @@
     <constant name="struts.excludedPackageNames"
               value="
                 ognl.,
+                java.io.,
```

Comment:
If you're considering including `java.io`, maybe `java.net` and `java.nio` should also be included in the default exclusions ?